spec/models/note_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+describe Note do
+
+  describe ".grouped_by_month" do
+    before do
+      @user = FactoryGirl.create(:user)
```

Comment:
use let instead of instance variables, [see betterspecs](http://www.betterspecs.org/#let)